modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1905,6 +2091,31 @@ private Expression<?> _getFunctionExpression(
 		throw new IllegalArgumentException("Invalid function " + function);
 	}
 
+	private Predicate _getLeftJoinLocalizationTablePredicate(
+			DynamicObjectDefinitionLocalizationTable
+				dynamicObjectDefinitionLocalizationTable,
+			DynamicObjectDefinitionTable dynamicObjectDefinitionTable)
+		throws PortalException {
+
+		if (!FeatureFlagManagerUtil.isEnabled("LPS-146755") ||
+			(dynamicObjectDefinitionLocalizationTable == null)) {
+
+			return null;
+		}
+
+		User user = GuestOrUserUtil.getGuestOrUser();
```

Comment:
In my opnion it's because it's the same method used [here](https://github.com/liferay/liferay-portal/blob/c38dc1e094f4dbfd1936e4c704776bd19e8331ac/portal-kernel/src/com/liferay/portal/kernel/service/BaseServiceImpl.java#L53-L55) for ServiceImpl, [for example](https://github.com/liferay/liferay-portal/blob/f6d10ea9c662b8d5408636c2805681d249dd0a1a/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java#L103)